.woodpecker.yml

```diff
@@ -204,7 +204,7 @@ steps:
       - bash api_tests/prepare-drone-federation-test.sh
       - cd api_tests/
       - yarn
-      - yarn api-test
+      - tail -f /tmp/lemmy*.out & yarn api-test
```

Comment:
Tried this locally, its extremely noisy with lots of (expected) error messages. Dont think it makes sense in CI.

Reply:
i know.. i only added it temporarily to try and figure out why the ci is failing (is there any other way to get access to the db or similar in woodpecker?

Reply:
I dont think so. You could try running the api tests manually on a small vps or a vm limited to 2 cores.